src/PrestaShopBundle/Form/Admin/Type/DeltaQuantityType.php

```diff
@@ -58,6 +68,10 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'constraints' => [
                     new Type(['type' => 'numeric']),
                     new NotBlank(),
+                    new Range([
+                        'min' => StockSettings::INT_32_MAX_NEGATIVE * 2, //because stock_mvt doesn't use negative numbers it can save 2x the size.
+                        'max' => StockSettings::INT_32_MAX_POSITIVE * 2,
```

Comment:
```suggestion
                        'min' => StockSettings::INT_32_MAX_NEGATIVE - StockSettings::INT_32_MAX_POSITIVE,
                        'max' => - StockSettings::INT_32_MAX_NEGATIVE + StockSettings::INT_32_MAX_POSITIVE,
```

Reply:
Because of the `-1` offset on `INT_32_MAX_POSITIVE` I'm not sure we can compute this range by simply multiplying it by 2 anymore So performing an addition of the two max values should be more accurate and represents the max values used for delta quantity when you want to edit from the max value to the min one (and vice versa)